charts/osm/templates/fluentbit-configmap.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.OpenServiceMesh.enableFluentbit}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fluentbit-configmap
+  namespace: osm-system
+data:
+  fluent-bit.conf: |-
+    [SERVICE]
+      Flush             5
+      Daemon            off
+      Log_Level         info
+      Parsers_File      parser.conf
+    [INPUT]
+      Name    tail
+      Tag     kube.*
+      Path    /var/log/containers/osm-controller-*_osm-system_osm-controller-*.log
```

Comment:
would this path change if the namespace in which osm is deployed changes ?

Reply:
yes, good catch!